meteor/client/ui/Shelf/EndWordsPanel.tsx

```diff
@@ -73,10 +73,56 @@ export class EndWordsPanelInner extends MeteorReactComponent<
 
 export const EndWordsPanel = translateWithTracker<IEndsWordsPanelProps, IState, IEndsWordsPanelTrackedProps>(
 	(props: IEndsWordsPanelProps) => {
-		const { activePieceInstance } = getIsFilterActive(props.playlist, props.showStyleBase, props.panel)
-		return { livePieceInstance: activePieceInstance }
+		return { livePieceInstance: getPieceWithManus(props) }
 	},
 	(_data, props: IEndsWordsPanelProps, nextProps: IEndsWordsPanelProps) => {
 		return !_.isEqual(props, nextProps)
 	}
 )(EndWordsPanelInner)
+
+function getPieceWithManus(props: IEndsWordsPanelProps): PieceInstance | undefined {
+	const currentPartInstanceId: any = props.playlist.currentPartInstanceId
+
+	const unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet = getUnfinishedPieceInstancesReactive(
+		props.playlist,
+		props.showStyleBase
+	)
+	let highestStartedPlayback: Time = 0
+	const startedPiecesCount: number = 0
+	unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet.forEach((pieceInstance: PieceInstance) => {
+		if (pieceInstance.startedPlayback && pieceInstance.startedPlayback > highestStartedPlayback) {
+			highestStartedPlayback = pieceInstance.startedPlayback
+		}
+	})
+
+	const unfinishedPieces = unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet.filter(
+		(pieceInstance: PieceInstance) => {
+			if (startedPiecesCount > 1 && pieceInstance.startedPlayback != highestStartedPlayback) {
+				return false
+			}
+			return !pieceInstance.startedPlayback || pieceInstance.startedPlayback == highestStartedPlayback
+		}
+	)
+
+	const activeLayers = unfinishedPieces.map((p) => p.piece.sourceLayerId)
+	const hasAdditionalLayer: boolean = props.panel.additionalLayers?.some((s) => activeLayers.includes(s)) || false
```

Comment:
```suggestion
	const hasAdditionalLayer: boolean = props.panel.additionalLayers?.some((s) => activeLayers.includes(s)) ?? false
```
Use nullish coalescing instead.

Reply:
It doesn't work if I use ??

Reply:
I just tried using `??` locally with the queue SOFIE.DEPLOY-TEST from zero. No problems. What were the issues?

Reply:
It was showing endword on everything even the layers not included in the filter